src/components/RessourceMenu/RessourceMenu.tsx

```diff
@@ -1,47 +1,74 @@
 import { useStore } from '../../store/store';
+
 import './RessourceMenu.scss';
 
+
+
+//const test = Math.floor(ressources.gold)
+
 export const RessourceMenu = () => {
   const ressources = useStore((state) => ({ ...state.ressources }));
   const buildingOutputs = useStore((state) => state.buildingOutputs);
   const reset = useStore((state) => state.reset);
 
+
+  const valueSmoothener: (m: number) => number = function (
```

Comment:
could you give the parameters meaningful names? makes the code easier to read 🙂 
same for the remainingRessource function